tests/run-make/dep-info/rmake.rs

```diff
@@ -0,0 +1,36 @@
+// This is a simple smoke test for rustc's `--emit dep-info` feature. It prints out
+// information about dependencies in a Makefile-compatible format, as a `.d` file.
```

Comment:
Suggestion: we should make it clear that we don't attempt to check the `.d` file is in fact Makefile-compatible in this test.

Reply:
@rustbot author